src/qibolab/pulses.py

```diff
@@ -1802,8 +1813,19 @@ def get_qubit_pulses(self, *qubits):
 
         new_pc = PulseSequence()
         for pulse in self.pulses:
-            if pulse.qubit in qubits:
-                new_pc.add(pulse)
+            if not isinstance(pulse, CouplerFluxPulse):
+                if pulse.qubit in qubits:
+                    new_pc.add(pulse)
+        return new_pc
+
+    def get_coupler_pulses(self, *couplers):
```

Comment:
```suggestion
    def coupler_pulses(self, *couplers):
```

Part of my personal fight against `get_` :P

Reply:
Should I be consistent and rename the `get_qubit_pulses` and` get_channel_pulses` ?

Reply:
Fine by me, eventually I definitely want to. However, this might even be postponed.

Reply:
I will remove the coupler one for now, in case I break stuff.